src/strategyHelpers/deploymentHelper.ts

```diff
@@ -59,16 +59,17 @@ export async function deployManifests(
             core.getInput('route-method', {required: true})
          )
 
-         const {result, newFilePaths} = await Promise.resolve(
+         const {result, newObjectsList} = await Promise.resolve(
             (routeStrategy == RouteStrategy.INGRESS &&
-               deployBlueGreenIngress(kubectl, files)) ||
+               (deployBlueGreenIngress(kubectl, files))) ||
```

Comment:
this has an extra () around deployBlueGreenIngress. I think it makes it harder to read